src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar__start-day--sun calendar__month-length--31">
```

Comment:
Fix BEM naming. `start-day` is a modifier, not element

```suggestion
      <div class="calendar calendar--start-day--sun calendar--month-length--31">
```